spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua

```diff
@@ -37,25 +38,26 @@ dao_helpers.for_each_dao(function(kong_config)
 describe("Admin API", function()
   local client
   local config_db
```

Comment:
It is unused, not sure why the linter didn't catch it. Didn't catch it locally either